drivers/fakenetepdriver.go

```diff
@@ -60,12 +60,12 @@ func (d *FakeNetEpDriver) DeleteMaster(node core.ServiceInfo) error {
 	return core.Errorf("Not implemented")
 }
 
-// AddBgpNeighbors is not implemented.
-func (d *FakeNetEpDriver) AddBgpNeighbors(id string) (err error) {
+// AddBgp is not implemented.
+func (d *FakeNetEpDriver) AddBgp(id string) (err error) {
 	return core.Errorf("Not implemented")
```

Comment:
reminds me that the unit tests for all this code is missing; more importantly system tests are missing too...
let's talk offline on how to incorporate testing of all this.